knowlg-automation/terraform/aws/eks_addon.tf

```diff
@@ -0,0 +1,7 @@
+resource "aws_eks_addon" "addons" {
+  for_each                 = {for addon in var.eks_addons : addon.name => addon }
+  cluster_name             = aws_eks_cluster.eks_cluster.id
+  addon_name               = each.value.name
+  addon_version            = each.value.version
+  resolve_conflicts_on_update = "PRESERVE"
```

Comment:
Can we use `resolve conflicts on create` ? Nevertheless we are going to deploy while creating the cluster.

Reply:
yes I think we can use ` resolve_conflicts_on_create = "OVERWRITE"` but this is more supposed as the updating of the config changes to add-on We can retain them with the usage of this `resolve_conflicts_on_update = "PRESERVE"`